staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/leaselock.go

```diff
@@ -89,6 +89,11 @@ func (ll *LeaseLock) Identity() string {
 	return ll.LockConfig.Identity
 }
 
+// returns the leader election config of the lock
+func (ll *LeaseLock) LeaderElectionConfig() (*int, *int, *int) {
+	return &ll.LockConfig.LeaseDurationSeconds, &ll.LockConfig.RenewDeadlineSeconds, &ll.LockConfig.RetryPeriodSeconds
```

Comment:
oh, now I get this. I don't like it at all to open up internals like that. Who locks those values?

If we want a way to change those values, add a setter method.